src/appComponents/common/ApiJsonFactory.js

```diff
@@ -9,7 +9,7 @@ var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
     'event/' + config.eventId;
         return {
             getJson: function ($apiEndpoint) {
-                let endpoint;
+                var endpoint;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'endpoint' should be initialized on declaration.](https://www.codacy.com/app/triplez/open-event-webapp/file/2423693653/issues/source?bid=3233857&fileBranchId=3238420#l12)